modules/apps/foundation/contacts/contacts-web/src/main/java/com/liferay/contacts/web/notifications/ContactsCenterUserNotificationHandler.java

```diff
@@ -108,6 +102,16 @@ protected String getBody(
 				new Object[] {creatorUserName});
 		}
 
+		if ((socialRequest.getStatus() !=
+				SocialRequestConstants.STATUS_PENDING) ||
+			(socialRequest.getCreateDate() >
+				userNotificationEvent.getTimestamp())) {
+
+			return StringUtil.replace(
+				_BODY_TEMPLATE_DEFAULT, new String[] {"[$BODY$]", "[$TITLE$]"},
```

Comment:
It seems the title would be empty unless socialRequest.getType() ==				SocialRelationConstants.TYPE_BI_CONNECTION.

That would mean that this markup would be completely empty. Shouldn't the code provide a default title for other types of requests?